lineman/app/components/contact_page/contact_page.haml

```diff
@@ -3,12 +3,12 @@
 
   .form-group
     %label{for: 'contact-name-field', translate: 'contact_message_form.name_label'}
-    %input#contact-name-field.form-control{ng-model: 'message.name', ng_required: 'true', placeholder: '{{contact_message_form.name_placeholder | translate}}'}
+    %input.form-control#contact-name-field{ng-model: 'message.name', ng_required: 'true', placeholder: '{{contact_message_form.name_placeholder | translate}}'}
 
     %label{for: 'contact-email-field', translate: 'contact_message_form.email_label'}
-    %input#contact-email-field.form-control{ng-model: 'message.email', ng_required: 'true', placeholder: '{{contact_message_form.email_placeholder | translate}}'}      
+    %input.form-control#contact-email-field{ng-model: 'message.email', ng_required: 'true', placeholder: '{{contact_message_form.email_placeholder | translate}}'}
 
     %label{for: 'contact-message-field', translate: 'contact_message_form.message_label'}
-    %textarea#group-description-field.form-control{ng-model: 'message.message', placeholder: '{{contact_message_form.message_placeholder | translate}}'}
+    %textarea#contact-message-field{ng-model: 'message.message', placeholder: '{{contact_message_form.message_placeholder | translate}}'}
```

Comment:
it's a public page .. try with a private browser. it's linked from the
homepage footer

On Mon, 24 Aug 2015 at 09:18 HSalmon <notifications@github.com> wrote:

> In lineman/app/components/contact_page/contact_page.haml
> <https://github.com/loomio/loomio/pull/2427#discussion_r37710654>:
>
> >
> >      %label{for: 'contact-message-field', translate: 'contact_message_form.message_label'}
> > -    %textarea#group-description-field.form-control{ng-model: 'message.message', placeholder: '{{contact_message_form.message_placeholder | translate}}'}
> > +    %textarea#contact-message-field{ng-model: 'message.message', placeholder: '{{contact_message_form.message_placeholder | translate}}'}
>
> That link doesn't work for me in angular
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/2427/files#r37710654>.
>